spec/factories/vm_azure.rb

```diff
@@ -1,7 +1,7 @@
 FactoryGirl.define do
   factory :vm_azure, :class => "ManageIQ::Providers::Azure::CloudManager::Vm", :parent => :vm_cloud do
     location "westus"
-    vendor   "azure"
+    vendor   "microsoft"
```

Comment:
@bzwei I think this is exposing a different bug.  The `:vendor` column is not being set consistently in the `refresh_parser.rb` and the line you referenced above is just wrong.  For a VM it is being set to "Microsoft" and for a Template it is being set to "microsoft".  The column is expected to be the lower case value and the vm_or_template.rb has a method that overrides the column and returns the value from the `VENDOR_TYPES` constant.
```ruby
  def vendor
    v = read_attribute(:vendor)
    VENDOR_TYPES[v]
  end
```
There is also a validation at the vm_or_template.rb layer that looks like it should fail because of this as well:
```ruby
validates_inclusion_of    :vendor, :in => VENDOR_TYPES.values
```

@blomquisg @djberg96 @bronaghs Can someone comment on the expected value for vendor here?  Either azure or microsoft?